src/things.ts

```diff
@@ -111,6 +111,29 @@ export const Things = {
 		}
 		return Labels.Localize(thing.label);
 	},
+	DrawWiki: (thing: Thing): HTMLElement => {
+		const wiki = document.createFullElement('div', {class: 'wiki'});
+		const link = document.createFullElement('a', {href: Router.GetWikiUrl(thing), target: '_blank'});
```

Comment:
"target: blank" is very bad for the user experience

Reply:
Open for discussion, for me it makes no sense to lose focus of the companion app.